src/app/pages/events/sfd-2024/components/growing-circles/growing-circles.component.html

```diff
@@ -0,0 +1,5 @@
+<div *ngFor="let size of nCircles; let i = index"
```

Comment:
Tal vez seria buena idea utilizar los nuevos fors de angular, que son tipo:

@for()...

Aunque a mi me dio problema cuando los estaba usando, asi que mejor lo dejamos asi.

Reply:
Mirare como funcionan, no los he probado.